apps/scan/frontend/src/screens/invalid_card_screen.tsx

```diff
@@ -1,11 +1,17 @@
+import { InsertedSmartCardAuth } from '@votingworks/types';
 import { InvalidCardScreen as SharedInvalidCardScreen } from '@votingworks/ui';
+
 import { ScreenMainCenterChild } from '../components/layout';
 
-export function InvalidCardScreen(): JSX.Element {
+export function InvalidCardScreen({
+  authStatus,
+}: {
+  authStatus: InsertedSmartCardAuth.LoggedOut;
+}): JSX.Element {
   return (
     <ScreenMainCenterChild infoBarMode="pollworker">
       <SharedInvalidCardScreen
-        reason="invalid_user_on_card"
+        reasonAndContext={authStatus}
```

Comment:
We'll actually now get better error messages on VxScan not just for jurisdiction mismatches but also other cases, e.g. election mismatches